media-video/mplayer/mplayer-1.4-r1.ebuild

```diff
@@ -234,6 +234,8 @@ src_prepare() {
 				"${FILESDIR}"/ffmpeg44.patch
 		has_version '>=media-video/ffmpeg-5' && eapply \
 				"${WORKDIR}"/${P}-ffmpeg5.patch
+		# Patch the configure script for the RISC-V architecture
+		eapply "${FILESDIR}/add-configure-riscv-support.patch" # bug 812855
```

Comment:
I'd suggest puttng the comments into the patch file, same with bug id, so make the ebuild simple and clean

also, I'd think it should also work with : PATCHES=(  "${FILESDIR}/add-configure-riscv-support.patch" ) ? but it's kind of different taste of style, so I'm fine with this

Reply:
use PATCHES= in new